osbuild/mounts.py

```diff
@@ -123,8 +123,11 @@ def __init__(self, args):
         self.mountpoint = None
         self.check = False
 
+    # pylint: disable= no-self-use
```

Comment:
Why the sudden whitespace between `=` and `no-self-use? My eyyyyyeeees! :eyes:

Reply:
Autope8 left it as is.

Reply:
I don't really mind tbh, but I'll fix it for your eyes ;)